src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -34,6 +34,12 @@ class BaseExpensiTextInput extends Component {
         this.onBlur = this.onBlur.bind(this);
     }
 
+    componentDidMount() {
+        if (this.props.autoFocus && this.input) {
```

Comment:
```suggestion
        // We are manually managing focus to prevent this issue: https://github.com/Expensify/App/issues/4514
        if (this.props.autoFocus && this.input) {
```